src/s2wasm.h

```diff
@@ -491,8 +494,11 @@ class S2WasmBuilder {
       skipWhitespace();
       if (match(".p2align")) s = strchr(s, '\n');
       else if (match(".int32")) {
-        initializerFunctions.emplace_back(cleanFunction(getStr()));
-        assert(implementedFunctions.count(initializerFunctions.back()));
+        do {
```

Comment:
we need to break at some point. can we remove white (*s) and use mustMatch(".p2align") before the code you suggested ?